tests/x11regressions/groupwise/groupwise.pm

```diff
@@ -0,0 +1,36 @@
+# Groupwise tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Groupwise client check
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use strict;
+use base "x11regressiontest";
+use testapi;
+use utils;
+
+sub run() {
+    select_console('root-console');
+    pkcon_quit;
+
+    # add repository and install groupwise
+    zypper_call("ar http://download.suse.de/ibs/SUSE:/Factory:/Head:/Internal/standard/ groupwise_repo");
```

Comment:
Unfortunately, groupwise client has SUSE-NonFree license, so there is no way to get it to public repository for openSUSE(or not?).  Also, it is just repackaged another rpm file with jar files, without any source compilation.  SUSE:Factory:Head:Internal repository (maintained by brave volunteers) is the only repository recommended by the company to employees on wiki https://wiki.microfocus.net/index.php?title=GroupWise_Linux .
It means that we are in "best effort" situation and we would like to have at least basic test coverage.